fiftyone/utils/torch.py

```
@@ -2630,18 +2630,36 @@ def from_image_classification_dir_tree(dataset_dir):
 
 
 def _load_image(image_path, use_numpy, force_rgb):
```

Ruff (0.14.13)

Missing return type annotation for private function `_load_image`

(ANN202)